packages/test-utils/src/snapshot-serializer.ts

```diff
@@ -0,0 +1,30 @@
+// Adapted from `jest-serializer-vue`
+// May be subject to changes due to Vue 3 core changes
```

Comment:
Adapted this into our local code. In the future we may perhaps modify it. Not sure if this is a good idea. Mostly because of changes to the `isVueInstance` signature changes in VueTestUtils and Vue 3 code.